include/QBDI/Memory.h

```diff
@@ -171,6 +212,31 @@ QBDI_EXPORT void qbdi_simulateCallV(GPRState *ctx, rword returnAddress, uint32_t
  */
 QBDI_EXPORT void qbdi_simulateCallA(GPRState *ctx, rword returnAddress, uint32_t argNum, const rword* args);
 
+/*! Get a list of all the memory maps (regions) of a process.
+ *
+ * @param[in]  pid  The identifier of the process.
+ * @param[out] size Will be set to the number of strings in the returned array.
+ *
+ * @return  An array of MemoryMap object.
+ */
+QBDI_EXPORT MemoryMap* qbdi_getRemoteProcessMaps(rword pid, size_t* size);
```

Comment:
This method should be removed as only used in the validator. Thus, the implementation should be in the validator